package.json

```diff
@@ -45,6 +46,7 @@
     "rimraf": "^2.4.4",
     "socket.io": "^2.1.0",
     "spawn-args": "^0.2.0",
+    "spdy": "^3.4.7",
```

Comment:
Right yeah I am under the mind that we should be using the built in modules for this instead of using express, which has had an open RB for almost 3 years for this support.